ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml

```diff
@@ -18,6 +18,20 @@
                     cq:policy="hclecomm/components/structure/search/policy_1589453975454"
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="wcm/core/components/policies/mapping"/>
+                <hclecomm jcr:primaryType="nt:unstructured">
+                    <components jcr:primaryType="nt:unstructured">
+                        <content jcr:primaryType="nt:unstructured">
+                            <button
+                                cq:policy="hclecomm/components/content/button/policy_1588076880690"
```

Comment:
@Rajamouli-hcl : why is there a new policy for button component ?

Reply:
removed button policy